addons/website_sale/static/src/snippets/s_add_to_cart/options.js

```diff
@@ -127,22 +127,13 @@ options.registry.AddToCart = options.Class.extend({
         const variantIds = this._variantIds();
         const buttonEl = this._buttonEl();
 
-        let productVariantId = variantIds[0];
-        buttonEl.dataset.visitorChoice = false;
-
-        if (variantIds.length > 1) {
-            // If there is more than 1 variant, that means that there are variants for the product template
-            // and we check if there is one selected and assign it. If not, visitorChoice is set to true
-            if (this.$target[0].dataset.productVariant) {
-                productVariantId = this.$target[0].dataset.productVariant;
-            } else {
-                buttonEl.dataset.visitorChoice = true;
-            }
-        }
-        buttonEl.dataset.productVariantId = productVariantId;
+        buttonEl.dataset.productTemplateId = this.$target[0].dataset.productTemplate;
+        // We're aware that getting the product id this way is too simplistic, but it mimics the
+        // previous logic. We'll fix this later on.
```

Comment:
So we don't forget :p
```suggestion
        // TODO LOTI VCR We're aware that getting the product id this way is too simplistic, but it mimics the
        // previous logic. We'll fix this later on.
```

Reply:
Done, although we should probably rewrite this whole file :sweat_smile: